documentation/book/ref-sample-kafka-resource-config.adoc

```diff
@@ -61,7 +61,8 @@ spec:
       lowercaseOutputName: true
       rules: <16>
       # Special cases and very specific rules
-      - pattern : kafka.server<type=(.+), name=(.+), clientId=(.+), topic=(.+), partition=(.*)><>Value
+      - pattern : kafka.server<type=(.+), name=(.+), clientId=(.+),
```

Comment:
I do not think this line should be wrapped. I think it will cause it to be misconfigured / invalid YAML.